main.lua

```diff
@@ -395,6 +416,19 @@ local function render_map()
     print("\u{2518}")
 end
 
+local function render_stats()
```

Comment:
normalna funkcja wyswietlania stat robi to automatycznie, ale takie podejscie tez przejdzie

Reply:
Nie do końca rozumiem co właściwie robi automatycznie...

Reply:
No iteruje na tablicy "PLAYER" i wyswietla staty, ale takie statyczne podejscie daje nam troche wiecej mozliwosci, wiec niewiem czy nie zmienie tamtej funkcji (np ta automatyczna wyswietla max hp jako: "max_hp" zamiast "Max HP")

Reply:
Właśnie mi się wydaje, że te podejście jest spoko. Ogólnie ta zmiana jest tylko podstawą na tworzenie display ze statystykami, dalej można sprawdzać statystyki komendą ale raczej usunąłbym tą możliwość w najbliższej przyszłości na rzecz UI

Reply:
usuniecie komendy na rzec UI brzmi dobrze